src/components/calculator.js

```diff
@@ -0,0 +1,45 @@
+import React, { Component } from 'react';
+
+export default class Welcome extends Component {
```

Comment:
- [ ] I would suggest you to name this component `Calculator` instead of ~`Welcome`~ in order to have readable, clean, and debbugable codebase
```suggestion
export default class Calculator extends Component {
```

Reply:
Done and resolved